lib/ansible/playbook/base.py

```diff
@@ -440,7 +440,17 @@ def get_validated_value(self, name, attribute, value, templar):
         if attribute.isa == 'string':
             value = to_text(value)
         elif attribute.isa == 'int':
-            value = int(value)
+            msg = ''
+            if isinstance(value, integer_types):
```

Comment:
There's no need to use `integer_types` over `int` since we don't support Python 2.